mcp-server/coneshare_mcp/tools/documents.py

```
@@ -47,3 +47,49 @@ async def delete_document(
+        file_size: int = Field(default=0, description="Size of the uploaded file in bytes"),
+        content_type: str = Field(default="application/octet-stream", description="MIME type of the file"),
+        path: Optional[str] = Field(default=None, description="Optional root-relative destination folder path"),
+    ) -> dict:
+        """Finalize document creation after streaming file content to the pre-signed upload URL.
+
+        Commit document metadata, create workspace records, and trigger background preview processing.
+        """
+        target_unique_name = unique_name or file_name
+        if not target_unique_name:
+            raise ValueError("Either unique_name or file_name must be provided to finalize document upload.")
+        return await client.finalize_document_upload(
```

Ruff (0.16.1)

[error] Undefined name `client`

(F821)